### ISSUE TITLE
Training And Fine Tuning Configuration

### ISSUE THREAD

**Issue by MarkWijkhuizen** [mit-han-lab/efficientvit#8]
Could you provide the training and fine tuning configuration, I would be especially interested in:
* Learning rate
* Optimizer
* Batch size
* Number of training epochs
* Data augmentations

**Comment by han-cai** [comment]
The training guide for ImageNet classification is available at https://github.com/mit-han-lab/efficientvit/blob/master/TRAINING.md.

The training guide for segmentation will be released soon.

Thanks for your interest in our work.
